README.md

```diff
@@ -1,5 +1,15 @@
 [![nginxconfig.io](public/assets/img/github.png)](https://nginxconfig.io)
 
+Fork adds a button to copy the generated config to the buffer as base64 string
```

Comment:
This was supposed to be displayed in the fork, but I do not think it is needed for the main repo.

Reply:
true